.travis.yml

```diff
@@ -3,7 +3,8 @@ language: node_js
 # test on node.js versions
 node_js:
   - 8
-  - 9
+  - 10
+  - 12
```

Comment:
Why this change?

Reply:
check if CI still works on these node versions